### DISCUSSION TITLE
I would like to see all features of LayeredNavigation by SwissLabs work

### DISCUSSION THREAD

**Discussion by websuits** [breezefront/community#4] [3. Integrations]
I would like to see all features of LayeredNavigation by SwissLabs work including Infinite Loading

**Comment by vovayatsyuk** [comment]
Thanks. We'll definitely add infinite loading integration in the next layered navigation update.

**Reply by websuits** [reply]
@vovayatsyuk can we create somehow a roadmap or sort of an ETA for this kind of implementation ? I've requested this feature by ticket and the sales team recommended me to open a specific implementation request, then i've been redirected to my own topic here, but i would like to find out more details.

**Reply by vovayatsyuk** [reply]
Infinite scroll and load more buttons are working already. AjaxLayerednavigation release will be tagged in one or two days.

**Comment by vovayatsyuk** [comment]
Just tagged the new release of ALN with Breeze integrations for infinite scroll and "More Products" button. https://docs.swissuplabs.com/m2/extensions/ajaxlayerednavigation/changelog/#version-1360

**Reply by websuits** [reply]
Thanks !
